src/PIL/Image.py

```diff
@@ -1627,7 +1660,9 @@ def apply_transparency(self):
 
         del self.info["transparency"]
 
-    def getpixel(self, xy):
+    def getpixel(
+        self, xy: tuple[SupportsInt, SupportsInt]
```

Comment:
The test suite does provide a list for `xy`
https://github.com/python-pillow/Pillow/blob/95a69ec6989deda8deb55b6fe842205327de9ca6/Tests/test_image_access.py#L217-L219

Also, why use `SupportsInt`, and not just `int`?